keyboards/rose75/info.json

```diff
@@ -0,0 +1,125 @@
+{
+    "manufacturer": "Matthijs Muller",
+    "keyboard_name": "ROSE75",
+    "maintainer": "Matthijs Muller",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "usb": {
+        "vid": "0x5363",
+        "pid": "0x7503",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "rows": ["GP27", "GP26", "GP25", "GP24", "GP18", "GP22"],
+        "cols": ["GP23", "GP21", "GP20", "GP19", "GP28", "GP12", "GP0", "GP1", "GP2", "GP3", "GP4", "GP5", "GP6", "GP7", "GP10"]
+    },
+    "ws2812": {
+        "pin": "GP29",
+        "driver": "vendor"
+    },
+    "rgb_matrix": {
+        "driver": "ws2812",
+        "layout": [
+            { "flags": 8, "matrix": [ 3, 0 ],  "x": 0,   "y": 0 },
+            { "flags": 8, "matrix": [ 3, 0 ],  "x": 15,  "y": 15 }
```

Comment:
You should not have duplicate matrix items here

Reply:
My bad, appreciate the review! As they are just 2 indicators, I labeled them as 0,0, and 0,1 for now.